usaspending_api/download/filestreaming/csv_source.py

```diff
@@ -61,7 +61,7 @@ def columns(self, requested):
         """Given a list of column names requested, returns the ones available in the source"""
         result = self.human_names
         if requested:
-            result = [header for header in requested if header in self.human_names]
+            result = [header for header in requested if header in self.human_names] or result[:1]
```

Comment:
So if the requested fields don't match any of the available csv column names, return the first key in self.human_names?

Reply:
Correct. In almost all cases that is the unique key. For some of the downloads we have multiple id columns and so I wasn't sure which to use. Elected to simply grab the first.

Reply:
We'll see what Ross' response is to this. I understand the justification. If it remains as-is, a comment describing "why" is a good idea